app/views/books/index.html.erb

```diff
@@ -24,12 +22,12 @@
     <% @books.each do |book| %>
       <tr>
         <td><%= book.title %></td>
-        <td><%= book.publisher_id %></td>
+        <td><%= book.publisher.name if book.publisher %></td>
```

Comment:
instead of the `if` add a function to the `book` model named `publisher_name`

Reply:
Okey